### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Create module
 - Create Readme.md

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+resource "aws_sns_topic_subscription" "this" {
+  count                  = var.enabled ? 1 : 0
+  endpoint               = var.subscription_endpoint
+  endpoint_auto_confirms = var.endpoint_auto_confirms
+  protocol               = var.subscription_endpoint_protocol
+  topic_arn              = aws_sns_topic.this[0].arn
+}
+
+resource "aws_cloudwatch_metric_alarm" "this" {
+  count                  = var.enabled ? 1 : 0
```
Comment (DimmKirr) [inline main.tf]: minor: indent is a bit off

**Review by DimmKirr — CHANGES_REQUESTED** [review]
Minor comment

**Comment by kobrikx** [comment]
[Create monitoring alert for AWS Budget anomaly detector](https://trello.com/c/8TYmmT9w/180-create-monitoring-alert-for-aws-budget-anomaly-detector)

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,123 @@
+variable "enabled" {
+  type        = bool
+  default     = true
+  description = "Gives ability to enable or disable a module"
+}
+
+variable "env" {}
+
+variable "aws_account_id" {
```
Comment (DimmKirr) [inline variables.tf:9]: Do we need to use a variable or we should rather use data?

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  description = "Endpoint endpoint for SNS topic subscription, PagerDuty (https://events.pagerduty.com/integration/<Integration Key>/enqueue)"
+}
+
+variable "subscription_endpoint_protocol" {
+  default     = "https"
+  description = "Endpoint protocol for SNS topic subscription"
+}
+
+variable "cost_filters_service" {
+  description = "Budget service cost filter, eg: Amazon Elastic Compute Cloud - Compute / Amazon Relational Database Service / Amazon Redshift / Amazon ElastiCache/ Amazon Elasticsearch Service"
```
Comment (DimmKirr) [inline variables.tf]: Please add a comment with a link to documentation for this parameter (to aws website, possibly?)

**Review by DimmKirr — CHANGES_REQUESTED** [review]
Looks good overall. But I'd add more links to the official aws documentation.

File: variables.tf (hunk 3 above)
Comment (kobrikx) [reply variables.tf]: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-what-is.html

File: README.md [hunk 4]
```diff
@@ -1 +1,100 @@
...
+### Minimal setup usage with email address
+
+```hcl
+module "budget_alert" {
+  source = "hazelops/budget-alert/aws"
+  env = var.env
+  limit_amount = "200"
+  subscription_endpoint_protocol = "email"
+  subscription_endpoint = "YourEmailAddress"
+  time_period_start = "2021-01-01_00:00"
```
Comment (DimmKirr) [inline README.md:36]: No closing bracket here

**Review by DimmKirr — APPROVED** [review]
Looks good, besides typo in README.

**Pull request merged**